CHANGELOG.md

```diff
@@ -1,16 +1,32 @@
 ## Unreleased
 
+FEATURES:
+* Support Kubernetes health probe synchronization with Consul for connect injected pods via `connectInject.healthChecks` [[GH-651](https://github.com/hashicorp/consul-helm/pull/651)].
+  The default behavior for this feature is `enabled: true`.
+  See [https://www.consul.io/docs/k8s/health](https://www.consul.io/docs/k8s/health) for more information.
+  **It is recommended to enable TLS with this setting enabled.**
```

Comment:
Maybe a comment on why it's recommended to enable TLS would be helpful, or a link to where in the doc it would describe that. So that I could know which network connections are insecure if I don't turn on TLS.

Reply:
doc in link ^ describes it

Reply:
I think we should still document it here since it only requires a few more words ("because the health checks controller makes calls to Consul agents across all nodes so ACL tokens could be stolen if an attacker was watching the network")

Reply:
Or your excellent description in values.yaml
```
  # NOTE: It is highly recommended to enable TLS with this feature because it requires
  # making calls to Consul clients across the cluster. Without TLS enabled, these calls
  # could leak ACL tokens.
```